### DISCUSSION TITLE
v1.0.2

### DISCUSSION THREAD

**Discussion by anitaparmar26** [codescandy/Dash-UI#12] [General]
- **Updated All plugins to latest**
- **Updated Docs**
- **Added Bootstrap 5 Components**
  - Offcanvas
  - Close Button
- **Added Utilities**
  - Colord Links
  - Ratio
  - Stacks
  - Text
  - Text Truncation
  - Vertical rule
  - Opacity
- **Updated Gray colors**

<hr /><em>This discussion was created from the release <a href='https://github.com/codescandy/Dash-UI/releases/tag/'>v1.0.2</a>.</em>

**Comment by ArthurD** [comment]
Hi!  

Love the project, don't mean to complain, just wanted to let you know/ask if I'm missing something before I jump to adding it as an issue.  :)

Upon running `npm audit` I'm seeing:
`16 vulnerabilities (8 moderate, 8 high)`

Running `npm audit fix` does not resolve the issue, and trying the same with the `--force` flag fails with an error, as well. 

Any thoughts, or is this something specific to versions I'm running perhaps?  I followed install instructions precisely & am using the latest code as of the time of writing.  

NPM version:  8.1.3
Node version:  17.0.1

Thank you & keep up the great work! :)

**Reply by anitaparmar26** [reply]
Hi 

16 vulnerabilities issue due to glob-parent not updating in package-lock.json file.
We solved this by below step:
1. Put after dependencies in package.json file.
  
   "resolutions": {
        "glob-parent": "^6.0.1"
    }

   and add also in package.json file.
  
   "scripts" {
     "preinstall": "npx npm-force-resolutions"
        }


2. After run "npx npm-force-resolutions" in terminal where your project runing


3. npm install

4. npm audit
5. run the task as "gulp"
